### PR TITLE
Unit test get_universe_versions() for CCXT

### PR DESCRIPTION
**Pull request by zli00185**
#311

### CODE REVIEW

**Comment by DanilYachmenev** [comment]
@zli00185 Should I review it?

**Comment by samarth9008** [comment]
> @zli00185 Should I review it?

There is nothing to review. Not even sure why this PR was generated as the file changes are also not related.

File: im_v2/ccxt/universe/test/test_universe.py [hunk 1]
```diff
@@ -46,3 +46,27 @@ def test_get_vendor_universe_as_full_symbol(self) -> None:
         self._test_get_vendor_universe_as_full_symbol(
             "CCXT", ["binance::BTC_USDT", "gateio::XRP_USDT", "kucoin::ETH_USDT"]
         )
+
+
+class TestGetUniverseVersions1(imvcountt.TestGetUniverseVersions1_TestCase):
+    def test_get_universe_versions_trade(self) -> None:
+        """
+        Check the universe version when mode is trade
```
Comment (samarth9008) [inline im_v2/ccxt/universe/test/test_universe.py:54]: -> `Test the universe versions when mode is trade.`

File: im_v2/ccxt/universe/test/test_universe.py [hunk 2]
```diff
@@ -46,3 +46,27 @@ def test_get_vendor_universe_as_full_symbol(self) -> None:
...
+        vendor = "CCXT"
+        mode = "trade"
+        expected_universes_str = (
+            "['v1', 'v2', 'v3', 'v4', 'v5', 'v6', 'v7', 'v7.1', 'v7.2', 'v7.3']"
+        )
+        self._test_get_universe_versions(vendor, mode, expected_universes_str)
+
+    def test_get_universe_versions_download(self) -> None:
+        """
+        Check the universe version when mode is download
```
Comment (samarth9008) [inline im_v2/ccxt/universe/test/test_universe.py:65]: -> `Test the universe versions when mode is download.`

**Review by samarth9008 — COMMENTED** [review]
Lets see if @DanilYachmenev has better docstrings. Also all your comments should end with a `.`.

File: im_v2/ccxt/universe/test/test_universe.py [hunk 3]
```diff
@@ -46,3 +46,27 @@ def test_get_vendor_universe_as_full_symbol(self) -> None:
         self._test_get_vendor_universe_as_full_symbol(
             "CCXT", ["binance::BTC_USDT", "gateio::XRP_USDT", "kucoin::ETH_USDT"]
         )
+
+
+class TestGetUniverseVersions1(imvcountt.TestGetUniverseVersions1_TestCase):
+    def test_get_universe_versions_trade(self) -> None:
+        """
+        Test the universe versions when mode is trade.
```
Comment (DanilYachmenev) [inline im_v2/ccxt/universe/test/test_universe.py:54]:
```
"""
Verify that trade universe versions are returned correctly.
"""
```

and similar but for download below

File: helpers/lib_tasks_utils.py
[the remark this answers is not in the document]
Comment (DanilYachmenev) [reply helpers/lib_tasks_utils.py:1]: still there

[the remark this answers is not in the document]
Comment (DanilYachmenev) [reply helpers/lib_tasks_utils.py:1]: Check yourself here

Снимок экрана 2023-07-06 в 03 22 13

[the remark this answers is not in the document]
Comment (zli00185) [reply helpers/lib_tasks_utils.py:1]: Done. I thought the delete was deleting the commit of change to the file and not the file itself.

File: helpers/lib_tasks_utils.py [hunk 4]
```diff
@@ -154,7 +154,6 @@ def _to_single_line_cmd(cmd: Union[str, List[str]]) -> str:
     return cmd
 
 
-# TODO(Grisha): make it public #755.
```
Comment (DanilYachmenev) [inline helpers/lib_tasks_utils.py:157]: Though still - why this change is here? it is not related to the issue, isnt it?

Comment (zli00185) [reply helpers/lib_tasks_utils.py:157]: I think it was the first contributor who made this change. Yea I should delete it anyway. This change doesn't make any sense.

**Review by DanilYachmenev — COMMENTED** [review]
lgtm
